WebAssets/tests/editing.tests.ts

```diff
@@ -1,8 +1,13 @@
-const TestDriver = require('./test-driver.js');
+import { TestDriver } from './test-driver';
 
 // TODO: remove in year 3000 when TC39 finally specs this

```

Comment:
Coming soon

Reply:
Seems like there was no progress since 2018
![image](https://user-images.githubusercontent.com/166396/77204449-9d15cd80-6b57-11ea-9fd7-deb4e15b5802.png)